Content.Client/Radio/EntitySystems/RadioJammerComponent.cs

```diff
@@ -0,0 +1,8 @@
+using Content.Shared.RadioJammer;
+
+namespace Content.Client.RadioJammer;
```

Comment:
```suggestion
namespace Content.Client.Radio.Components;
```
I also forgot to add this, if you do plan on keeping the components separate on the client and server then you should correct this namespace.

Reply:
I'll be honest I don't quite know why I did it this way vs the other way your talking about. I looked how other objects did something similar and all the ones I looked at did separate client and server components so I just copied that. I'll try doing this change as I don't really have time to figure out the shared component thing. If you think its important it be shared let me know and it will take a while but I'll figure it out 😆

Reply:
Usually most stuff should go in shared unless it needs to do something only the server/client can do or if you want to hide information like hiding certain antagonist components from the client. Either way I'm not a maintainer, and until one says otherwise you can make this however you work. It does work after all.